lerna.json

```diff
@@ -4,5 +4,5 @@
   ],
   "npmClient": "yarn",
   "useWorkspaces": true,
-  "version": "1.0.0"
+  "version": "1.0.1"
```

Comment:
Is that an intended change?

Reply:
It was changed by running `yarn run lerna version 1.0.1 --no-git-tag-version --no-push` which is suggested in Polkadot, rather than manually changed.